src/types/Todo.ts

```diff
@@ -0,0 +1,6 @@
+export interface Todo {
+  id: number;
```

Comment:
The indentation seems to be off. Make sure to use soft tabs set to 2 spaces. You can adjust your editor settings to do this automatically. 😊